.gitignore

```diff
@@ -154,4 +154,4 @@ cython_debug/
 
 # Custom Ignores
 data/*
-./app/uploads/*
```

Comment:
Your repo is very difficult to navigate as there are 2,040 files in the changeset, stalling the website.
Most of these files are unnecessary.
Add static/eng-char to .gitignore